agent/php_execute.c

```diff
@@ -2027,7 +2035,7 @@ static void nr_php_instrument_func_end(NR_EXECUTE_PROTO) {
 
   wraprec = segment->wraprec;
 
-  if (wraprec && wraprec->is_exception_handler) {
+  if (wraprec && segment->is_exception_handler) {
```

Comment:
does it need the wraprec check in the `if` statement anymore?

Reply:
It probably doesn't. `segment->is_exception_handler` can only be set if wraprec for user exception handler has been found in `nr_php_instrument_func_begin`.

Reply:
Simplified the check in 5c7f58da1820dad00dfd8c0125f08467328fb8d7.